plugins/kafka/kafka.cpp

```diff
@@ -1031,9 +1033,10 @@ namespace KafkaPlugin
 // Plugin Initialization and Teardown
 //==============================================================================
 
-#define CURRENT_KAFKA_VERSION "kafka plugin 1.0.0"
+#define CURRENT_KAFKA_VERSION "kafka plugin 1.1.0"
 
 static const char* kafkaCompatibleVersions[] = {
+    "kafka plugin 1.0.0",
```

Comment:
I am unsure if changes made in this PR resulted in an incompatible module or not.  If it is incompatible, this line should be removed.  Feedback welcome.

Reply:
To be honest the compatible versions mechanism is pretty redundant these days, as we ship plugins in lock-step with platform versions and expect them to match.